doc/modules/gaussian_process.rst

```diff
@@ -608,9 +608,25 @@ References
       Link to an official complete PDF version of the book 
       `here <http://www.gaussianprocess.org/gpml/chapters/RW.pdf>`_ . 
 
-.. currentmodule:: sklearn.gaussian_process
+Select Dimension Kernel
+------------------
+
+The :class:`SelectDimensionKernel` is thin wrapper around a given kernel. It
```

Comment:
Can you write a line or two about why this is useful. For instance, if different features follow different assumptions?